app/assets/javascripts/challenges.js

```diff
@@ -1,2 +1,12 @@
 // Place all the behaviors and hooks related to the matching controller here.
```

Comment:
There is a js -> coffeescript converter online, can you go ahead and run that on this file? Only the application.js should not have a `.coffee` extension.